### DISCUSSION TITLE
Oracle DB Integration - DONE

### DISCUSSION THREAD

**Discussion by barnadsv** [Budibase/budibase#2285] [Ideas]
Hello, Any chance of integration with Oracle DB?

**Comment by joebudi** [comment]
Hey @barnadsv 

I must admit, I thought this request would come sooner. In September, we might have time to look at adding 1 or 2 additional integrations/data sources. I will discuss with the team, but I can't promise anything right now.

**Reply by barnadsv** [reply]
I'm glad you want to add new integrations!
Maybe I'm lucky ..
Congratulations on the excellent product!

**Comment by Oen1997** [comment]
Hi, 

Installed Budi, started playing with such a nice tool, and discovered that Oracle support is missing ;)
Is it comming soon?

**Reply by mjashanks** [reply]
All I can say is that it's slowly rising up the list of priorities. I can't commit to anything right now, but there's a good chance that we will get it done this year.

**Comment by mjashanks** [comment]
@barnadsv  @Oen1997  Oracle integration is done.

**Reply by barnadsv** [reply]
Great news!

**Reply by Oen1997** [reply]
Awesome!
